src/tools/xkbctrl

```diff
@@ -116,6 +88,9 @@ sub ReadDataConfigMap {
 		die "could not open file: $filename";
 	}
 	while (my $line=<DATA>) {
+		$line =~ s/\t+/ /g;
+		$line =~ s/ +/ /g;
+		$line =~ s/ /:/g;
```

Comment:
in general logic is to replace all tabs and spaces together with one colong so it can look like:
```
$line =~ s/[ \t]+/:/g;
```
single line is then enough